Emergence/Contexts/Main Menu/Getting Shows/ShowEmitter.swift

```diff
@@ -11,6 +11,14 @@ protocol ShowEmitter {
     func showAtIndexPath(index: NSIndexPath) -> Show
     func getShows()
     func onUpdate( callback: EmitterUpdateCallback )
+    
+    func isEqualTo(show: ShowEmitter) -> Bool
+}
+
+extension ShowEmitter {
```

Comment:
my first idea as well but I basically ran into this: http://www.llamagraphics.com/content/combining-swift-protocols